spec/lib/tasks/mou_signatures.rake_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rake"
+
+require "rails_helper"
+
+RSpec.describe "mou_signatures.rake" do
+  before do
+    # Rake.application.options.trace = true
```

Comment:
Don't think you meant to leave this in?

Reply:
I copied this from the other rake task test. Have removed here and in the file I copied from too.